tests/unit/cli/cli-test.js

```diff
@@ -133,6 +145,29 @@ describe('Unit: CLI', function() {
     td.verify(init(), {ignoreExtraArgs: true, times: 0});
   });
 
+  describe('process signal listeners', function() {
+    var onCommandInterrupt;
+    beforeEach(function() {
+      onCommandInterrupt = td.matchers.isA(Function);
+    });
+
+    it('sets up signal listeners before command started', function() {
+      stubValidateAndRun('serve', function() {
```

Comment:
Can we verify that the handler is added before the command is invoked?  I don't think it's critical, but it would be a nice improvement to the test if it doesn't cause too much churn elsewhere in the file